wagtail/images/formats.py

```diff
@@ -1,26 +1,37 @@
+from warnings import warn
+
 from django.utils.html import escape
 from django.utils.translation import gettext_lazy as _
 
 from wagtail.utils.apps import get_app_submodules
+from wagtail.utils.deprecation import RemovedInWagtail60Warning
 
 from .shortcuts import get_rendition_or_not_found
 
 
 class Format:
-    def __init__(self, name, label, classnames, filter_spec):
+    def __init__(self, name, label, classname, filter_spec):
         self.name = name
         self.label = label
-        self.classnames = classnames
+        self.classname = classname
         self.filter_spec = filter_spec
```

Comment:
I have not added support for more complex customisations where maybe `self.classnames` is set in an sub-classed `__init__`. I started doing this and it felt too complex for an undocumented use case of 'maybe users maybe will do this in a really odd way'.

Happy to add this if we feel it's needed for something as widespread as the Image Format code.

Reply:
I feel that should be enough.

Reply:
Thanks.